app/src/main/kotlin/io/element/android/x/initializer/TracingInitializer.kt

```diff
@@ -71,7 +71,7 @@ class TracingInitializer : Initializer<Unit> {
         return WriteToFilesConfiguration.Enabled(
             directory = bugReporter.logDirectory().absolutePath,
             filenamePrefix = "logs",
-            filenameSuffix = null,
+            filenameSuffix = "log",
```

Comment:
Add a small comment so we don't change this?

Reply:
Yeah, good call.